### DISCUSSION TITLE
Проксация только определенных ресурсов

### DISCUSSION THREAD

**Discussion by TheSainEyereg** [xtrime-ru/antizapret-vpn-docker#63] [Q&A] [answered]
Я хочу на роутере организовать связку zapret + antizapret т.е. чтобы сайты заблокированные шли через средство обхода DPI, а те ресурсы которые недоступны для адресов РФ шли через antizapret. Как можно это организовать?

**Comment by xtrime-ru** [comment]
Расскажите подробнее про zapret? Это docker контейнер?

**Reply by TheSainEyereg** [reply]
Нет, это способ обхода DPI. Как GoodbyeDPI, только с большим функционалом и работает на всех платформах в т.ч. на роутерах. Конкретно в Linux он ставится как сервис

https://github.com/bol-van/zapret

**Comment by TheSainEyereg** [comment, marked as answer]
Переменную `SKIP_UPDATE_FROM_ZAPRET` установить на `true`
README: https://github.com/xtrime-ru/antizapret-vpn-docker?tab=readme-ov-file#environment-variables
